src/components/header/Header.js

```diff
@@ -19,32 +21,69 @@ function Header() {
           <span className="navicon"></span>
         </label>
         <ul className="menu">
-          <li>
-            <a href="#skills">Skills</a>
-          </li>
-          { exp === true &&
-            <li>
-              <a href="#experience">Work Experiences</a>
-            </li>
-          }
-          <li>
-            <a href="#opensource">Open Source</a>
-          </li>
-          <li>
-            <a href="#projects">Projects</a>
-          </li>
-          <li>
-            <a href="#achievements">Achievements</a>
-          </li>
-          <li>
-            <a href="#blogs">Blogs</a>
-          </li>
-          <li>
-            <a href="#talks">Talks</a>
-          </li>
-          <li>
-            <a href="#contact">Contact Me</a>
-          </li>
+          {(() => {
+            if (skillsSection) {
+              return (
+                <li>
+                  <a href="#skills">Skills</a>
+                </li>
+              );
+            }
+          })()}
+          {(() => {
+            if (openSource.display) {
+              return (
+              <li>
+                <a href="#opensource">Open Source</a>
+              </li>
+              );
+            }
+          })()}
```

Comment:
Hey nice catch on these conditional rendering! 

What about this shorter version of code:
```
{openSource.display &&
   <li>
      <a href="#opensource">Open Source</a>
    </li>
}
```

Reply:
mmmm i don't really think about that but its a good point maybe i change when resolve merge conflicts. :+1:

Reply:
@Ergouzii we have merged #92 can you resolve conflicts now

Reply:
@saadpasta Hey I'm not the author of this PR so I can't resolve the conflicts on my side.

Reply:
Ah, I think he meant to ping @exspiravit.